crates/ark/src/modules/rstudio/stubs.R

```diff
@@ -45,11 +45,12 @@
     height <- .ps.validate.viewer.height(height)
 
     # Derive a title for the viewer from the path.
+    normalizedPath <- normalizePath(url, mustWork = FALSE)
     title <- .ps.viewer.title(normalizedPath)
 
     invisible(.Call("ps_html_viewer",
         url,     # The URL of the file to view
```

Comment:
```suggestion
    url <- normalizePath(url, mustWork = FALSE)
    
    # Derive a title for the viewer from the path.
    title <- .ps.viewer.title(url)

    invisible(.Call("ps_html_viewer",
        url,     # The URL of the file to view
```

The main advantage of my tweak is that we now also pass the normalized `url` to Rust as well, which feels right, and is what we've done elsewhere too